README.md

```diff
@@ -6,7 +6,7 @@
 <a href="https://youtube.com/narlock"><img src="https://raw.githubusercontent.com/narlock/TamoStudy/main/README%20Assets/youtubeApp.png" alt="YouTube" align="right" width="60"></a>
 <a href="https://tamostudy.com"><img src="https://raw.githubusercontent.com/narlock/TamoStudy/main/Application%20Assets/icon_512x512%402x.png" alt="TamoStudy logo" title="TamoStudy" align="right" width="60"></a>
 
-# TamoStudy
+# TamoStudy --- Spanish Translation
```

Comment:
If you are interested in creating a spanish translation of the readme, can you make a README_ES.md file and translate the readme?

This line needs to be remove from this

Reply:
yeah ngl i forgot i wrote that in. wrote that in when i was figuring out wtf i was doing LMAO.